assets/css/_street_view_button.scss

```diff
@@ -1,7 +1,24 @@
 .m-street-view-button {
+  /* Layout */
+  /** Prevent the button from being inlined */
+  display: block;
+  /** Prevent the button from expanding to available space */
+  width: fit-content;
+
+  /* Element Decoration */
   background-color: black;
-  color: white;
-  padding: 0.25rem;
   border-radius: 0.5rem;
+  padding: 0.25rem;
+
+  /* Text Decoration */
+  color: white;
   text-decoration: none;
+  line-height: 1;
+}
+
+/* Override `.leaflet-container a` */
+// REVIEWERS: Please Pick One
+.leaflet-container .m-street-view-button,
+:root .m-street-view-button {
+  color: white;
```

Comment:
I'd say given that Leaflet is the reason that this is in here, `.leaflet-container .m-street-view-button` makes more sense as a selector.